### ISSUE TITLE
Beeps should be toggleable

### ISSUE THREAD

**Issue by ultrasound1372** [mltony/nvda-browser-nav#9]
I rarely interact with sites where BrowserNav's parent/child navigation is necessary, that is I don't often read reddit or hacker news, or much other commenting sites like that. As such, the beeps this add-on presents are mostly extraneous information, and on sites like this one actually can be somewhat painful, given how high the beeps get with some of the supposed levels around nav bars. I understand these are to convey the level relationship. But is it possible to have a gesture to disable this indication while still allowing the roter navigation? Perhaps it could be baked into site settings, so you could, say, turn beeps off globally but then turn them back on on some sites where they would be useful. At the moment I have to cludge this by setting beep volume to 0, which is a global setting and does not go through site settings in the slightest. @mltony

**Comment by mltony** [comment]
Yes, this should be  doable. You're right, currently you can only 
disable beeps globally by setting their volume to zero. I'll add this to 
the roadmap.

**Comment by ultrasound1372** [comment]
@mltony Any progress on this?

**Comment by ultrasound1372** [comment]
@mltony Has this been implemented, thus meaning we can close this issue? Or has it still not been done yet?

**Comment by Neurrone** [comment]
This is definitely still an issue in the latest release. I just installed this add-on and was wondering why there is so much noise when navigating websites.
